QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -28,13 +32,20 @@
 
 	private static double π = Math.PI;
 	
+	private static FloatStatus centricAngleOffset;
+	private static FloatStatus calibratedAngle = new FloatStatus(0);
+	private static BooleanStatus fieldCentric = new BooleanStatus();
+	
 	private static EventOutput mecanum = new EventOutput() {
 		public void event() {
+			float speed;
+			float rotationspeed;
+			double angle;
+			
```

Comment:
Any particular reason that you moved these up to the top? It's usually clearer in-line.